tests.py

```diff
@@ -33,12 +33,12 @@ def test_bank_webcash_string_serialization(self):
         amount = 500
         secret_value = "feedbeef"
         n1 = SecretWebcash(amount, secret_value)
-        self.assertEqual(str(n1), f"e{amount}:secret:{secret_value}")
```

Comment:
So the decimals are always there now? Why?

Reply:
why not truncate trailing zeroes?

Reply:
I don't have strong arguments either way, but fwiw:
- I think (a couple of people in the Discord seemed to agree) that always having 8 decimals makes some secrets easier to read, because you can count zeros from the end (compare `0.000001` vs `0.00000100`).
- Truncating the zeros adds another operation to the formatting of amounts (negligible, I know).
- Personally I prefer the visual consistency of same-length strings vs variable length.

As I said none of these are very strong reasons - but they are _something_. Do you have a stronger case for truncating the zeros?

Reply:
has truncation been implemented? https://discord.com/channels/929473826633424986/929473826633424989/963803003385708615

Reply:
Not yet